docs/api/tools/change_headline.py

```diff
@@ -0,0 +1,28 @@
+"""
+Tool to help formatting the docs via sphinx autoapi
+"""
+
+
+import argparse
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='replace first two line')
+
+    parser.add_argument('files', type=str, nargs='*')
+    parsed_args = parser.parse_args()
+
+    n_files = 0
+    for rest_file in parsed_args.files:
+        with open(rest_file, 'r') as f:
+            first_word = f.readline().strip().split()[0]
+            if not '.' in first_word:
```

Comment:
__[Flake8]__

test for membership should be 'not in'


<!-- Comment by [SideCI](https://sideci.com) -->